src/UserStories/UserStories.css

```
@@ -8,31 +8,36 @@ html {
   overflow: hidden;
 }
 .stories-scrollable {
+  transition: transform 0.2s ease-out;
+  min-height: 100vh;
+  flex-wrap: nowrap;
   display: flex;
-  padding-left: 50vw;
-  transition: transform 0.5s ease-in-out;
-  height: 100vh;
+  align-items: center;
 }
 .stories-container {
-  margin: 70px auto;
-  height: 800px;
-  min-width: 449px;
+  margin: 18px 0;
   border-radius: 8px;
-  margin-left: 50px;
+  transition: all 0.5s ease-out;
```

I mean this, it should be faster than the story resize:
```suggestion
  transition: all 0.2s ease-out;
```

---

Yupe, much better